src/core/middleware.js

```diff
@@ -81,3 +81,15 @@ export function prefixMiddleWare(req, res, next, { _config = config } = {}) {
 
   return next();
 }
+
+export function trailingSlashesMiddleware(req, res, next) {
+  const URLParts = req.originalUrl.split('?');
```

Comment:
Do you know if `req.originalUrl` is URL encoded or is it already decoded? I like how simple your patch is but I wonder if we should go with something more battle tested like [url.parse()](https://nodejs.org/docs/latest/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost).